src/views/Home.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import Slider from "react-slick";
+import sliderData from '../mocks/en-us/featured-banners.json';
+import productCategoriesData from '../mocks/en-us/product-categories.json';
+import productsData from '../mocks/en-us/featured-products.json';
+import { sliderBannersSettings, sliderProductsSettings } from '../utils/constants.js';
+import Product from '../components/Product.js';
+
+class Home extends Component {
+    render() {
+        const sliderLayoutItems = sliderData.results.map((item) => {
+            return (
+                <div key={item.id}>
+                    <img src={item.data.main_image.url} alt={item.data.main_image.alt} className="wz-slider-item-image"/>
+                    <h3 className="wz-center">{item.data.title}</h3>
+                </div>
+            )
+        });
+
+        const productSliderItems = productCategoriesData.results.map((item) => {
+            return (
+                <div key={item.id}>
+                    <img src={item.data.main_image.url} alt={item.data.main_image.alt} className="wz-slider-product-item-image"/>
+                    <h3 className="wz-center">{item.data.name}</h3>
+                </div>
+            )
+        });
+
+        const productsDataItems = productsData.results.map((item) => {
```

Comment:
Creo que estos tres (sliderLayoutItems, sliderLayoutItems y productsDataItems) puede funcionarte bien si los divides en componentes mas pequeños, uno para cada uno. Así puedes llegar a reutilizar en alguna otra sección tal vez.

Reply:
The components mentioned above has been simplified and extracted to additional components. They're on the respective folder.